packages/kit/src/exports/vite/index.js

```diff
@@ -626,31 +624,6 @@ function kit({ svelte_config }) {
 		 * @see https://vitejs.dev/guide/api-plugin.html#configurepreviewserver
 		 */
 		configurePreviewServer(vite) {
-			// generated client assets and the contents of `static`
-			// should we use Vite's built-in asset server for this?
-			// we would need to set the outDir to do so
-			const { paths } = svelte_config.kit;
-			const assets = paths.assets ? SVELTE_KIT_ASSETS : paths.base;
-			vite.middlewares.use(
-				scoped(
-					assets,
-					sirv(join(svelte_config.kit.outDir, 'output/client'), {
-						setHeaders: (res, pathname) => {
-							if (pathname.startsWith(`/${svelte_config.kit.appDir}/immutable`)) {
-								res.setHeader('cache-control', 'public,max-age=31536000,immutable');
-							}
-							if (vite_config.preview.cors) {
-								res.setHeader('Access-Control-Allow-Origin', '*');
-								res.setHeader(
-									'Access-Control-Allow-Headers',
-									'Origin, Content-Type, Accept, Range'
-								);
-							}
```

Comment:
should this stuff be present in the green portion of the diff?

Reply:
No, it wasn't there in Kit 1. Vite will add it for us if we do it in the returned callback. I had to add it in Kit 2 because we were doing it before the Vite middlewares could add it